clubs/README.md

```diff
@@ -174,35 +190,194 @@ keep building their website to make it unique. At the second meeting, you'll
 
 Here's our suggested schedule:
 
 1. Wait for hackers to arrive (5 minutes)
 2. Announcements (5 minutes)
+3. Example demo (5 minutes)
 3. Free-form hacking (45 minutes)
-4. Demos (20 minutes)
+4. Demos (30 minutes)
+
+#### Wait for Hackers to Arrive
+
+This'll be very similar to the first meeting. Put on a playlist (not too loud)
+and have a slide on the projector to let people know they're in the right place.
+
+#### Announcements
+
+Announcements are your opportunity to share any updates with your members and
+let them know what they'll be doing in this meeting. In addition to sharing the
+meeting's schedule, they're useful for sharing upcoming events, upcoming guest
+visits, and any other information relevant to your members.
+
+Tell your members that for this meeting, they're going to continue where they
+left off and continue building their websites. 30 minutes before the end of the
+meeting, everyone's going to come up to the front and demo the websites they
+built. Right after announcements you're going to give an example of what a demo
+looks like.
+
+#### Example Demo
+
+You'll want to give an example of what a demo looks like so your members know
+what to expect at the end of the meeting. The best way to do this is to demo a
+project that you or another person built for the Portfolio workshop.
+
+If you don't have a website that you built to demo, go ahead and pull up one of
+the Portfolio workshop examples that were shown in the first meeting and use it.
+[Click here][demo] for an example of what a project demo looks like (this demo
+was given at a conference).
+
+[demo]: https://www.youtube.com/watch?v=MY01d647S9Y
+
+#### Free-Form Hacking
+
+This'll basically be the same as the free-form hacking section in the first
+meeting. Members will continue building their websites and your role is to
+create an environment where people are comfortable for asking for help.
+
+Just like in the first meeting, when people do have questions, be eager to help
+them out. It's also a good idea to put on an upbeat playlist (but again, not too
+loud -- people should feel comfortable talking with one another).
+
+#### Demos
+
+Before getting to the demos themselves, give your members 45, 30, 15, and 5
+minute warnings so the demos aren't a surprise.
+
+30 minutes before demos start, have everyone email you a link to their website.
+After making the announcement telling your member to do this, go to each person
+individually to confirm that they emailed you a link. If they run into any
+trouble, help them now. Hackers will be able to continue to add to their
+websites after they email you the link -- any changes they make will be live at
+the same URL they emailed you after they push to GitHub.
+
+Quick warning before we get into the nitty-gritty of running demos: these are
+hard to facilitate and you're going to have to be detail-oriented when running
+them.
+
+To run the demos themselves, pick a hacker to demo first and a direction to form
```

Comment:
Let's bias for merging for the time being. If you feel strongly about this, a pull request would be welcome :-).